src/membrane-keypad-element.ts

```diff
@@ -8,16 +8,23 @@ export class MembraneKeypadElement extends LitElement {
 
   private pressedKeys = new Set<string>();
 
+  isNumeric(text: string) {
```

Comment:
should be private (or you can just declare it as a plain local function at the start of the file, so it's not even a part of the class)